modules/ob-build-controller-metrics.adoc

```diff
@@ -0,0 +1,32 @@
+// This module is included in the following assembly:
+//
+// builds/build-controller-observability.adoc
+
+:_content-type: REFERENCE
+[id="ob-build-controller-metrics_{context}"]
+= Build controller metrics
+
+You can check the following build controller metrics for monitoring purposes:
+
+
+.Build controller metrics
+[options="header"]
+|===
+
+| Name | Type | Description | Status
```

Comment:
@shipsing any specific reason for skipping the `labels` section from the table?
ref: https://shipwright.io/docs/build/metrics/

Reply:
No specific reason. I think i missed out on that detail. I will add a label column as well.